sotodlib/core/metadata/obsdb.py

```diff
@@ -204,13 +228,29 @@ def update_obs(self, obs_id, data={}, tags=[],
             tag name is prefxed with '!', then the tag name will be
             un-applied, i.e. cleared from this observation.
 
-        Returns:
-            self.
+        Example of ways to pass updates to obsdb when there are multiple primary keys.
+        1) obs_id as str and wafer_info as tuple::
```

Comment:
Add a blank line above 1) so it renders properly ([docs build](https://github.com/simonsobs/sotodlib/pull/1159)).